docs/frontend.md

```diff
@@ -138,9 +136,9 @@ See [`@18f/identity-analytics` package documentation][analytics_package] for cod
 how to track an event in JavaScript.
 
 Any event logged from the frontend must be added to the `ALLOWED_EVENTS` allowlist in [`FrontendLogController`][frontend_log_controller.rb].
-This mapping associates the event name logged from the frontend with the corresponding method from
-[AnalyticsEvents][analytics_events.rb] to be called. All properties will be passed automatically to
-the event from the frontend as long as they are defined in the method argument signature.
+This is an allowlist of events defined in [AnalyticsEvents][analytics_events.rb] which are allowed
```

Comment:
does this URL need to point at the actual file?
```suggestion
This is an allowlist of events defined in [AnalyticsEvents][../app/services/analytics_events.rb] which are allowed
```

Reply:
This comment is making me realize that the way I went about this is prone to causing confusion, but these are using reference links, defined at the end of the heading section:

https://github.com/18F/identity-idp/blob/c7fa569251f696dc8d25799a9490bef207b5fed1/docs/frontend.md?plain=1#L152-L153